### PR TITLE
change fluxVariability as default for the sampling and the tutorials

### PR DESCRIPTION
**Pull request by Gpreciat**
The sampling function chrrSampler.m has as default function fluxVariability.m to perform FVA. fastFVA it is optional.

Adapted in the tutorial.

**I hereby confirm that I have:**

- [x] Tested my code on my own machine
- [x] Followed the guidelines in the [Contributing Guide](https://opencobra.github.io/cobratoolbox/docs/contributing.html)
- [x] Selected `develop` as a target branch (top left drop-down menu)

*(Note: You may replace [] with [X] to check the box)*

### CODE REVIEW

File: tutorials/uniformSampling/tutorial_uniformSampling.m [hunk 1]
```diff
@@ -91,12 +91,17 @@
 % The CHRR algorithm is an useful tool to sample high dimentional models (bigger 
 % than 10,000 reactions). 
 %% Equipment setup
+% To perform FVA using the function |fastFVA| it is necessary to have CPLEX 
+% solver installed. In this tutorial, we will use the function |fluxVariability|. 
+% Change the variable |options.useFastFVA = 1 |to use |fastFVA|.
 
- changeCobraSolver('gurobi');
+options.useFastFVA = 0;
```
Comment (laurentheirendt) [inline tutorials/uniformSampling/tutorial_uniformSampling.m]: This option `options.useFastFVA` should be an explicit boolean, i.e. `options.useFastFVA = false;`

File: tutorials/uniformSampling/tutorial_uniformSampling.m [hunk 2]
```diff
@@ -119,8 +124,13 @@
 % Flux variability analysis (FVA) returns the minimum and maximum possible flux 
 % through every reaction in a model.
 
-[minUn, maxUn] = fastFVA(unlimitedOx, 100);
-[minLim, maxLim] = fastFVA(limitedOx, 100);
+if options.useFastFVA
+    [minUn, maxUn] = fastFVA(unlimitedOx, 100);
+    [minLim, maxLim] = fastFVA(limitedOx, 100);
+else
+    [minUn, maxUn] = fluxVariability(unlimitedOx);
```
Comment (laurentheirendt) [inline tutorials/uniformSampling/tutorial_uniformSampling.m]: I would prefer to also pass on the option `useFastFVA` explicitly to `fluxVariability()`. @syarra?

File: src/analysis/sampling/CHRR/chrrSampler.m [hunk 3]
```diff
@@ -63,20 +63,24 @@
 else
     toGetWidths = 0;
 end
+
+if nargin<8 || isempty(useFastFVA)
+    useFastFVA=0;
```
Comment (laurentheirendt) [inline src/analysis/sampling/CHRR/chrrSampler.m]: Try using a true boolean, i.e. `useFastFVA = false`

File: src/analysis/sampling/CHRR/chrrSampler.m [hunk 4]
```diff
@@ -1,10 +1,10 @@
-function [samples, roundedPolytope, minFlux, maxFlux] = chrrSampler(model, numSkip, numSamples, toRound, roundedPolytope, minFlux, maxFlux)
+function [samples, roundedPolytope, minFlux, maxFlux] = chrrSampler(model, numSkip, numSamples, toRound, roundedPolytope, minFlux, maxFlux, useFastFVA)
```
Comment (laurentheirendt) [inline src/analysis/sampling/CHRR/chrrSampler.m]: As the signature changes, may you also update the docstring accordingly?

File: src/analysis/sampling/sampleCbModel.m [hunk 5]
```diff
@@ -57,6 +59,7 @@
 nFilesSkipped = 2;
 maxTime = 10 * 3600;
 toRound = 1;
+useFastFVA = 0;
```
Comment (Gpreciat) [inline src/analysis/sampling/sampleCbModel.m]: ok

File: tutorials/uniformSampling/tutorial_uniformSampling.m (hunk 1 above)
Comment (Gpreciat) [inline tutorials/uniformSampling/tutorial_uniformSampling.m]: ok

File: src/analysis/sampling/CHRR/chrrSampler.m (hunk 4 above)
Comment (Gpreciat) [inline src/analysis/sampling/CHRR/chrrSampler.m]: ok

File: src/analysis/sampling/CHRR/chrrSampler.m (hunk 3 above)
Comment (Gpreciat) [inline src/analysis/sampling/CHRR/chrrSampler.m]: ok

**Comment by laurentheirendt** [comment]
Closing temporarily as discussed with @Gpreciat.

**Pull request closed without merging**